_docs_v5/4_guides/scripting.md

```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
+| cmd_tlm_reload             | Command and Telemetry Server | Unimplemented |
+| display                    | Telemetry Viewer | Unimplemented |
+| get_all_packet_logger_info | Command and Telemetry Server | Deprecated |
+| get_background_tasks       | Command and Telemetry Server | Deprecated |
+| get_cmd_list               | Command and Telemetry Server | Deprecated, use get_all_commands |
+| get_cmd_log_filename       | Command and Telemetry Server | Deprecated |
+| get_cmd_param_list         | Command and Telemetry Server | Deprecated use get_command |
+| get_all_interface_info     | Command and Telemetry Server | Deprecated, use get_inter |
+| get_disconnected_targets   | Script Runner | Unimplemented |
+| get_interface_info         | Command and Telemetry Server | Deprecated, use get_interface |
+| get_interface_targets      | Command and Telemetry Server | Deprecated |
+| get_output_logs_filenames | Command and Telemetry Server | Deprecated |
+| get_packet_data | Command and Telemetry Server | Deprecated |
+| get_packet_logger_info | Command and Telemetry Server | Deprecated |
+| get_packet_loggers | Command and Telemetry Server | Deprecated |
+| get_replay_mode | Replay | Deprecated |
+| get_router_info | Command and Telemetry Server | Deprecated, use get_router |
+| get_screen_definition | Telemetry Viewer | Unimplemented |
+| get_screen_list | Telemetry Viewer | Unimplemented |
+| get_scriptrunner_message_log_filename | Command and Telemetry Server | Deprecated |
+| get_server_message | Command and Telemetry Server | Deprecated |
+| get_server_message_log_filename | Command and Telemetry Server | Deprecated |
+| get_server_status | Command and Telemetry Server | Unimplemented |
+| get_target_ignored_items | Command and Telemetry Server | Deprecated, use get_target |
+| get_target_ignored_parameters | Command and Telemetry Server | Deprecated, use get_target |
+| get_target_info | Command and Telemetry Server | Deprecated, use get_target |
+| get_tlm_details | Command and Telemetry Server | Deprecated |
+| get_tlm_item_list | Command and Telemetry Server | Deprecated |
+| get_tlm_list | Command and Telemetry Server | Deprecated |
+| get_tlm_log_filename | Command and Telemetry Server | Deprecated |
+| interface_state | Command and Telemetry Server | Deprecated, use get_interface |
+| local_screen | Script Runner | Unimplemented |
+| map_target_to_interface | Command and Telemetry Server | Unimplemented |
+| override_tlm_raw  | Command and Telemetry Server | Deprecated, use override_tlm |
+| open_file_dialog | Script Runner | Unimplemented |
+| open_files_dialog | Script Runner | Unimplemented |
+| open_directory_dialog | Script Runner | Unimplemented |
+| prompt_combo_box | Script Runner | Unimplemented |
+| prompt_message_box | Script Runner | Unimplemented |
+| prompt_to_continue | Script Runner | Unimplemented |
+| prompt_vertical_message_box | Script Runner | Unimplemented |
```

Comment:
I think prompt_* are all internal methods, not part of the user facing api.

Reply:
Will remove